chsdi/tests/mapproxy/test_wmtscapabilities.py

```diff
@@ -0,0 +1,112 @@
+#-*- coding: utf-8 -*-
+
+import sys
+import os
+import random
+from unittest import TestCase
+from testconfig import config as tc
+
```

Comment:
`import requests` is missing here, meaning you can't run chsdi/tests/mapproxy/test_wmtscapabilities.py alone.